src/common/components/ReviewItem/ReviewItem.tsx

```diff
@@ -0,0 +1,82 @@
+import { reviewItemTheme, type ReviewItemVariants } from "./ReviewItem.theme";
+import { ReviewBody } from "./ReviewBody";
+import { ProfileSchool } from "./ProfileSchool";
+import { ProfileReviewer } from "./ProfileReviewer";
+import { getHumanReadableTimestampDelta } from "@/common/functions";
+import { ThumbUpFilledIcon } from "@/common/components/CustomIcon";
+import { LockCtaOverlay } from "@/common/components/LockCtaOverlay";
+import { Button } from "@/common/components/Button";
+
+// TODO: to replace with prisma generated types
+export type ReviewLabel = {
+  name: string;
+  typeof: "professor" | "course";
+};
+
+export type Review = {
+  body: string;
+  courseCode: string;
+  username: string;
+  likeCount: number;
+  labels: ReviewLabel[];
+  createdAt: number;
+};
+
+export type ReviewItemProps = ReviewItemVariants & {
+  review: Review;
+  isLocked?: boolean;
+  variant?: "home" | "professor" | "course";
+};
+
+export const ReviewItem = ({
+  review,
+  isLocked,
+  variant = "home",
+}: ReviewItemProps) => {
+  const {
+    wrapper,
+    headingContainer,
+    metadataContainer,
+    likeIcon,
+    timedelta,
+    body,
+  } = reviewItemTheme();
+
+  return (
+    <div className={wrapper()}>
+      <div className={headingContainer()}>
+        {variant === "home" ? (
+          <ProfileSchool courseCode={review.courseCode} />
+        ) : (
+          <ProfileReviewer name={review.username} />
+        )}
+        <div className={metadataContainer()}>
+          {variant === "home" ? (
+            <ProfileReviewer name={review.username} />
+          ) : (
+            variant === "professor" && (
+              <ProfileSchool courseCode={review.courseCode} />
+            )
+          )}

```

Comment:
I think ternary condition might not be easy to read. How about something like this?

```suggestion
          {variant === "home" && (
            <ProfileReviewer name={review.username} />
          )}
           {variant === "professor" && (
              <ProfileSchool courseCode={review.courseCode} />
            )}
```

Reply:
sounds good! accepted suggestion at 5d75c58